### DISCUSSION TITLE
Plaid level (developer vs production)

### DISCUSSION THREAD

**Discussion by bpracht** [ebridges/plaid2qif#35] [Q&A]
I love this idea, but when I created a Plaid account, it want me to either do a developer version (limiting to 100 items (ie. transactions?)), or get a production account, but the request says it requires a compliance review, contract commitments and a 1+ day turnaround.

**Comment by csmwww** [comment]
Hi @ebridges @bhipple, who appear most active here:

Especially with the recent [death of Mint](https://robberger.com/mint-alternatives) following the death of [MS Money](https://gnucash.uservoice.com/forums/101223-feature-request/suggestions/2351447-cashflow-forecast-with-graph?page=1&per_page=20) your offering here seems to hold some promise for a vast number of lost souls ([relevance](https://news.ycombinator.com/item?id=37954403)).

Can you clarify a bit better the intended user of your efforts here?

You note [GNUCash](https://github.com/ebridges/plaid2qif/blob/master/README.md), which interests me as part of a personal solution, but like @bpracht I see obtaining a Plaid account appears to be "development only" and it is not clear there how one engages as a consumer (or even developer/personal use), with implications only institutional subscription might be offered through sales (implicitly $500 and up since a full developer setup seems to start at that price point).

So, who is intended to maybe use your fine work, who may actually be using it, and how are you personally using "[it with ~5-10 institutions without issue](https://github.com/ebridges/plaid2qif/discussions/37)"?

_To clarify: my interest at present seems to be like @bpracht, personal - for my own finance. I am not affiliated with any financial institution, working instead as a programmer buried in my own little world in the printing industry. Though, or course, I'd like to see something like your efforts here reach to helping lots of consumers like me, not just me, that are not finding useful solutions to survive our own financial chaos. You are certainly invited to reach out directly to me if you think it more appropriate - crismooney.com._

**Reply by ebridges** [reply]
Hi @csmwww --

At this stage there is no intended user from my standpoint as this project has been deprecated (see [#36](https://github.com/ebridges/plaid2qif/issues/36)for more details).

I use GNUCash and the QIF files always imported fine.  I did not test it with other products, so YMMV.

Ultimately I've moved on from Plaid as it became apparent that our use cases did not align.

Good luck!
Ed

**Comment by csmwww** [comment]
Hi Ed:

Thank you for taking the time to respond here, and I apologize for not noticing the "issues" tab in GitHub and thus taking your time (I was just poking at "discussion").

Sad for you having spent that time, and sad for us to that Plaid did not work out - I suspect their product offereing/pricing structure is a concern here.  An old man like me has dozens of accounts (too many to be puling QIF/CSV files all the time) that I really need to monitor more real time, and I had fantasized your efforts would have helped me and others like me, maybe with just some addition of friendly docs and/or minor glue code additions for average users that might more actively bind plaid2qif to GNUCash. Thanks for trying.

Looks like Simplify for me (but they failed to connect to my bank "KeyBank" so now I'm on a support ticket already with them). Empower app is mediocre, though it connects to all my accounts. Tiller Money may be the best for us nerds. In all this, I might at least get data centralized vs hitting a couple dozen accounts, and your https://github.com/ebridges/csv2qif may be helpful (thank you for that effort as well).

Since I've been a programmer for 45 years and am getting close to retirement which may give me more time, if you happen to chat with Ben Hipple you might encourage him to reach to me to see if he might fill me in on details about relations with Plaid and any other options to se if I might help since GNUCash has served me well for my community organization efforts for years. I don't see this problem going away, but I also appreciate it may not be solvable with open source lack of funding either.

Good day sir,
Cris

**Comment by ebridges** [comment]
I believe @bhipple is in these discussions as well I f you’d like to coordinate directly with him.  That said, it’s based on a business decision from Plaid and not likely to be able to be worked around.  Best of luck

**Comment by bpracht** [comment]
Please keep me posted. FWIW, I was willing to pay some money, but at the
very least wanted to know how much.

Would anyone be interested in a Plaid competitor?
